docs/products/kafka/concepts/acl.md

```diff
@@ -1,116 +1,216 @@
+|                      | `Describe`              | [docs](https://kafka.apache.org/30/javadoc/org/apache/kafka/clients/admin/Admin.html#describeConsumerGroups(java.util.Collection)) | ✓         | ✓                        |             | ✓           | ✓              | ✓                    |
+|                      | `ListConsumerGroups`    | [docs](https://kafka.apache.org/30/javadoc/org/apache/kafka/clients/admin/Admin.html#listConsumerGroups(org.apache.kafka.clients.admin.ListConsumerGroupsOptions)) | ✓         | ✓                        |             | ✓           | ✓              | ✓                    |
+|  Topics            | `Read`                  | [docs](https://kafka.apache.org/30/javadoc/org/apache/kafka/clients/consumer/KafkaConsumer.html#poll(java.time.Duration))    | ✓         | ✓                        |             | ✓           | ✓              | ✓                    |
+|                      | `Write`                 | [docs](https://kafka.apache.org/30/javadoc/org/apache/kafka/clients/producer/KafkaProducer.html#send(org.apache.kafka.clients.producer.ProducerRecord,org.apache.kafka.clients.producer.Callback)) | ✓         | ✓                        | ✓           |             | ✓              | ✓                    |
+|                      | `Describe`              | [docs](https://kafka.apache.org/30/javadoc/org/apache/kafka/clients/admin/Admin.html#listTransactions())                    | ✓         | ✓                        | ✓           | ✓           | ✓              | ✓                    |
+|                      | `DescribeConfigs`       | [docs](https://kafka.apache.org/30/javadoc/org/apache/kafka/clients/admin/Admin.html#describeConfigs(java.util.Map))         | ✓         |                          |             |             |                | ✓                    |
+|                      | `AlterConfigs`          | [docs](https://kafka.apache.org/30/javadoc/org/apache/kafka/clients/admin/Admin.html#alterConfigs(java.util.Map))           | ✓         |                          |             |             |                | ✓                    |
+|                      | `Delete`                | [docs](https://kafka.apache.org/30/javadoc/org/apache/kafka/clients/admin/Admin.html#deleteTopics(java.util.Collection))     | ✓         |                          |             |             | ✓              | ✓                    |
+|  TransactionalId   | `DescribeTransactions`  | [docs](https://kafka.apache.org/30/javadoc/org/apache/kafka/clients/admin/Admin.html#describeTransactions(java.util.Collection)) | ✓         |                          | ✓           |             |                | ✓                    |
+|                      | `BeginTransaction`      | [docs](https://kafka.apache.org/30/javadoc/org/apache/kafka/clients/producer/KafkaProducer.html#beginTransaction())         | ✓         |                          | ✓           |             |                | ✓                    |
```

Comment:
Honestly I don't understand this table anymore. Old one tried to explain what Kafka opertions are allowed by different Aiven ACL rules. Columns Aiven ACLs and Kafka-native ACLs do not make sense to me in this context.

Reply:
Thanks. I reverted the table to the previous one.